### PR TITLE
ZENKO-3413: adjust date format for tables

### PR DESCRIPTION
**Pull request by theocerutti**
Date format updated from 
image
to
image

(only for tables)

Tables updated:
* AccountKeys / Created On
* Bucket Browsing / Created On

### CODE REVIEW

**Review by nicolas2bert — COMMENTED** [review]
What about object page's **Modified on** column?

**Comment by theocerutti** [comment]
> What about object page's Modified on column?

It already has the good format
image

**Review by nicolas2bert — APPROVED** [review]

**Review by Cuervino — CHANGES_REQUESTED** [review]
Looks good!

It was not in the requirement, but could you please make those 2 columns aligned to the right (as they are numerical)?
From the visual point of view, it would mainly change the header position, the content would not move.
image

File: src/react/databrowser/buckets/BucketList.jsx [hunk 1]
```diff
@@ -46,7 +46,12 @@ export default function BucketList({ selectedBucketName, buckets, locations }: P
         {
             Header: 'Created on',
             accessor: 'CreationDate',
-            Cell: ({ value }) => { return formatShortDate(new Date(value));},
+            headerStyle: { textAlign: 'right', paddingRight: '32px' },
+            Cell({ value }: { value: string }) { return (
+                <div style={{ textAlign: 'right', paddingRight: '16px' }}>
```
Comment (nicolas2bert) [inline src/react/databrowser/buckets/BucketList.jsx]: Can you also use the TextAligner component here?

**Review by Cuervino — APPROVED** [review]
LGTM!

**Review by nicolas2bert — APPROVED** [review]

**Comment by theocerutti** [comment]
@bert-e approve

**Pull request merged**